### DISCUSSION TITLE
Why use e.g. givenName not given_name as MobileConnect and OIDC define it

### DISCUSSION THREAD

**Discussion by AxelNennker** [camaraproject/KnowYourCustomer#158] [General]
OIDC and [MobileConnect](https://www.gsma.com/identity/wp-content/uploads/2023/01/IDY.23-v1.0.pdf) define claims.
OIDC defines standard claims and how to retrieve userinfo in https://openid.net/specs/openid-connect-core-1_0.html#Claims
The [OIDF WG on eKYC Identity Assurance](https://openid.net/wg/ekyc-ida/specifications/) relies on OIDC [claims](https://openid.bitbucket.io/ekyc/openid-connect-4-identity-assurance.html).

Why does CAMARA KYC use different name for the attributes/claims?

e.g. why use givenName when both OIDC and MobileConnect use given_name?

Should CAMARA use the "standard" claims? And add its own if needed?

**Comment by fernandopradocabrillo** [comment]
The use of camelCase for attributes was agreed by Commonalities for all CAMARA APIs at the beginning of the project. It is not something specific for KYC, it is in the [API Design Guidelines](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#42-inputoutput-resource-definition)
